jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTaskListGridPresenterTest.java

```diff
@@ -39,6 +41,7 @@
 import static org.dashbuilder.dataset.filter.FilterFactory.*;
 import static org.dashbuilder.dataset.sort.SortOrder.*;
 import static org.mockito.Mockito.*;
+import static org.junit.Assert.*;
 
 @RunWith(GwtMockitoTestRunner.class)
 public class DataSetTaskListGridPresenterTest {
```

Comment:
I think would be nice to have a task to check the columns displayed by table. That way we can be sure that all task variables are visible.